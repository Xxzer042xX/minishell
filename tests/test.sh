#!/bin/bash

cd `dirname "$0"`
cd ../
make

./minishell "VAR1=test1" "VAR2=test=2" "VAR1=" "=test1"
# > tests/test.out

cd tests/

diff test.out test.ref
