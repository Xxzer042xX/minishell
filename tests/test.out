print this string with newline
print this string without newlineprint this string \n with newline
/Users/vviterbo/Dropbox/Mac/Desktop/42/minishell
/Users/vviterbo/Dropbox/Mac/Desktop/42/minishell/tests/testdir
______
GREP_COLOR=1;33
__GIT_PROMPT_IGNORE_STASH=
TERM_PROGRAM=vscode
TERM=xterm-256color
SHELL=/bin/zsh
HISTSIZE=10000
CLICOLOR=1
TMPDIR=/var/folders/_s/21f6ng2s59x7m5zzk1pfkgkc0000gn/T/
CONDA_SHLVL=1
TERM_PROGRAM_VERSION=1.96.4
CONDA_PROMPT_MODIFIER=(base) 
ORIGINAL_XDG_CURRENT_DESKTOP=undefined
MallocNanoZone=0
OLDPWD=/Users/vviterbo/Dropbox/Mac/Desktop/42/minishell/tests
__GIT_PROMPT_SHOW_UPSTREAM=
USER=vviterbo
HISTFILESIZE=20000
__GIT_PROMPT_SHOW_UNTRACKED_FILES=all
COMMAND_MODE=unix2003
CONDA_EXE=/Users/vviterbo/miniconda3/bin/conda
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.SkMyJgD0RK/Listeners
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x12
_CE_CONDA=
LSCOLORS=GxFxCxDxBxegedabagaced
PATH=/opt/local/bin:/opt/local/sbin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/cmake/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/opt/X11/bin:/Library/Apple/usr/bin:/Users/vviterbo/miniconda3/bin:/Users/vviterbo/miniconda3/condabin
XML_CATALOG_FILES=file:///Users/vviterbo/miniconda3/etc/xml/catalog file:///etc/xml/catalog
CONDA_PREFIX=/Users/vviterbo/miniconda3
__CFBundleIdentifier=com.microsoft.VSCode
PWD=/Users/vviterbo/Dropbox/Mac/Desktop/42/minishell
LANG=fr_FR.UTF-8
VSCODE_GIT_ASKPASS_EXTRA_ARGS=
XPC_FLAGS=0x0
_CE_M=
XPC_SERVICE_NAME=0
HISTCONTROL=ignoredups:erasedups
SHLVL=3
HOME=/Users/vviterbo
VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js
GIT_BRANCH=main
__GIT_PROMPT_SHOW_CHANGED_FILES_COUNT=1
__GIT_PROMPT_IGNORE_SUBMODULES=
CONDA_PYTHON_EXE=/Users/vviterbo/miniconda3/bin/python
LOGNAME=vviterbo
VSCODE_GIT_IPC_HANDLE=/var/folders/_s/21f6ng2s59x7m5zzk1pfkgkc0000gn/T/vscode-git-7c27d38db8.sock
CONDA_DEFAULT_ENV=base
VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Plugin).app/Contents/MacOS/Code Helper (Plugin)
GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh
DISPLAY=/private/tmp/com.apple.launchd.WXfM3XApga/org.macosforge.xquartz:0
HISTTIMEFORMAT=%F %T 
COLORTERM=truecolor
_=./minishell
